include/stdafx.h

```diff
@@ -30,6 +30,13 @@ typedef std::complex<double> double_complex;
 typedef std::vector<double_complex> vec_complex;
 typedef std::vector<double> vec_double;
 
+#if (defined(__GNUC__) || defined(__GNUG__)) && !(defined(__clang__) || defined(__INTEL_COMPILER))
+typedef __float128 qp;
+//#include <quadmath.h>
```

Comment:
Does this need to be uncommented? Also, does `-lquadmath` need to be added to `setup.py`?